### PR TITLE
GF pagination/AJAX

### PR DESCRIPTION
**Pull request by karpstrucking**
Issue #132

### CODE REVIEW

**Comment by bmarshall511** [comment]
@karpstrucking @jaredatch Thanks for the work you guys put into this. @jaredatch Would you mind confirming this fix? I don't have/purchased Gravity Forms.

File: src/js/zerospam.js [hunk 1]
```diff
@@ -33,6 +32,17 @@
           return true;
         });
 
+       	// Gravity Forms
+       	$( document ).bind( "gform_post_render", function() {
```
Comment (naomicbush) [inline src/js/zerospam.js]: Nitpicky, but use [.on](http://api.jquery.com/on/) instead of .bind

**Comment by jaredatch** [comment]
@bmarshall511 sure thing

**Comment by jaredatch** [comment]
We don't need to bind to the submit and the render event, so we can drop the submit which simplifies it.

I need to actually test this after lunch, but I believe it should be able to be shrunk slightly to:

```
       	// Gravity Forms
       	$( document ).on( "gform_post_render", function() {
            $( "<input>" ).attr( "type", "hidden" )
                .attr( "name", "zerospam_key" )
                .attr( "value", zerospam.key )
                .appendTo( ".gform_wrapper form" );
       	});
```

**Comment by karpstrucking** [comment]
That works in my testing, will update.

**Comment by jaredatch** [comment]
Solid. Should be ready for @bmarshall511 to push upstream.

**Comment by bmarshall511** [comment]
Thanks everyone!

**Pull request merged**